.gitlab-ci.yml

```diff
@@ -394,3 +393,106 @@ drush-deploy-prod:
       when: manual
     - if: '$CI_COMMIT_BRANCH =~ /^release\//'
       when: manual
+
+deploy-straight-to-prod:
+  image: mcr.microsoft.com/azure-cli
+  stage: production
+  needs:
+    - job: build-and-push-images
+  environment:
+    name: production
+    url: https://intranet.essex.gov.uk/
+  before_script:
+    - az extension add --name containerapp
+    - az login --service-principal -u${ARM_CLIENT_ID} -p${ARM_CLIENT_SECRET} -t${ARM_TENANT_ID}
+  script:
+    - echo "Importing image from Pre-Production into Production"
```

Comment:
Should this be `"Importing image from Pre-Production into Production"`?

Reply:
Done